src/Appwrite/Utopia/Response.php

```diff
@@ -435,9 +435,13 @@ public function output(Document $document, string $model): array
                             throw new Exception('Missing model for rule: ' . $ruleType);
                         }
 
-                        $item = $this->output($item, $ruleType);
+                        $data[$key][$index] = $this->output($item, $ruleType);
                     }
                 }
+            } else {
```

Comment:
Why did we make this change? Is it part of this PR context?

Reply:
I'm not sure where this change has come from 🤔 Will investigate

Reply:
There are coming from `fix-unit-tests` which is already merged in `0.16.x`, shall I try remove? Or sync `0.16.x`?

Reply:
Sync with 0.16x